src/hugo/content/en-us/patterns/MVC/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Model-View-Controller Device Software Design"
+weight: 10
+aliases:
+  - /designs/MVC/
+summary: "recommendation for the design and structure of device software"
+---
+
+## Challenge
+
+IoT devices have a common set of concerns: 
+* report the current state of the device, attached sensors and actuators
+* processing control messages (both remote and local)
+* presenting status information 
+
+The design should also be easy to test and reuse across multiple projects, hardware platforms, languages, and SDKs. Some devices are 'one-way' only reporting state or taking action, while others are 'two-way,' performing both of these actions. The design should be generally composable to support all these uses as well as accommodate a flexible variety of peripherals, command sources, and interfaces. 
+
+## Solution
+
+An IoT device software design can be based on the [Model-View-Controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) design pattern.
+
+![Model-View-Controller Pattern](MVC-UML.png) 
+
+### Basic Model View Controller
+
+The **model** component's concern is to represent the overall state of the device: signals, metrics, peripherals, display data, and current state of the control variable(s). The **view** component handles the presentation of the data from the **model**. The **controller** is responsible for taking actions to change the state or configuration of the device and peripherals.
+
+### Basic Model View Controller Diagram Steps
+
+1. The **model** only reads from the device. And only the **model** is allowed to read from the device.
```

Comment:
Both sentences read the same? How about "The model, and only the mode, reads from the device."?